spec/facility_spec.rb

```diff
@@ -23,4 +30,131 @@
       expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
     end
   end
-end
+  
+  describe '#Default attributes' do
+    it 'can add vehicle registration' do
+      expect(@facility.services).to eq([])
+      @facility.add_service('Vehicle Registration')
+      expect(@facility.services).to eq(['Vehicle Registration'])
+     expect(@facility.registered_vehicles).to eq([])
+     expect(@cruz.registration_date).to eq(nil)
+     expect(@facility.collected_fees).to eq(0)
+   end
+ end
+
+  describe'#Register vehicle' do 
+    it 'Registers cruz' do 
+      @facility.add_service('Vehicle Registration')
+      @facility.register_vehicle(@cruz)
+      expect(@facility.registered_vehicles).to eq([@cruz])
+      expect(@cruz.registration_date).to eq(Date.today.year)
+      expect(@cruz.plate_type).to eq(:regular)
+      expect(@facility.registered_vehicles).to eq([@cruz])
+      expect(@facility.collected_fees).to eq(100)
+    end
+
+    it 'Registers camaro and bolt' do
+      @facility.add_service('Vehicle Registration')
+      @facility.register_vehicle(@cruz)
+      expect(@facility.registered_vehicles).to eq([@cruz])
+      expect(@cruz.registration_date).to eq(Date.today.year)
+      expect(@cruz.plate_type).to eq(:regular)
+      expect(@facility.registered_vehicles).to eq([@cruz])
+      expect(@facility.collected_fees).to eq(100)
+    
+      
+      @facility.register_vehicle(@camaro)
+      expect(@facility.registered_vehicles).to eq([@cruz, @camaro])
+      expect(@camaro.registration_date).to eq(Date.today.year)
+      expect(@camaro.plate_type).to eq(:antique)
+      expect(@facility.collected_fees).to eq(125)
+  
+      
+   
+      @facility.register_vehicle(@bolt)
+      expect(@facility.registered_vehicles).to eq([@cruz, @camaro, @bolt])
+      expect(@bolt.registration_date).to eq(Date.today.year)
+      expect(@bolt.plate_type).to eq(:ev)
+      expect(@facility.collected_fees).to eq(325)
+       
+    end
+  end
+
+  describe'#check facility and facility_2' do 
+    it 'sees what is in the first facility' do 
```

Comment:
This message could be more specific and technical. What exactly are we looking at here? We want each it block to test a single method, or in this case, looks like we're testing that a facitiliy can register vehicles.